### PR TITLE
updates Dockerfile so that R packages are installed

### PR DESCRIPTION
**Pull request by strangeties**

### CODE REVIEW

File: src/planscape/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,28 @@
-FROM python:3
+FROM rocker/tidyverse as final
```
Comment (riecke) [inline src/planscape/Dockerfile:1]: This seems to pull in a lot including RStudio (https://hub.docker.com/r/rocker/tidyverse). Is it necessary?

Comment (strangeties) [reply src/planscape/Dockerfile:1]: tidyverse will be necessary (and can take some time to install), but RStudio isn't necessary.

I looked into multistage builds briefly, but there isn't much documentation for copying tidyverse packages from a build image to a final image. I can try again.

image-size-wise, Django + R (installed through apt) was 1.83GB, while Django + rocker/tidyverse (obtained from a base image) + forsys was 2.78GB, which suggests there's some room for improvement.

**Review by riecke — APPROVED** [review]

**Comment by riecke** [comment]
Do we need this anymore, Wendy?

**Comment by strangeties** [comment]
I should convert this to a draft.
There were 2 follow-up todo's:
1) double-check the licensing for rocker/tidyverse and add the info to the security doc
2) see whether the R package size can be reduced via a multi-stage docker file

**Comment by runforyourlives** [comment]
This is obsoleted by our current architecture.

**Pull request closed without merging**